### DISCUSSION TITLE
Support for Additional Resources in SEB 3

### DISCUSSION THREAD

**Discussion by malone01** [SafeExamBrowser/seb-win-refactoring#427] [General]
Hello,
the "Support for Additional Resources (external and embedded), which can be opened with icons and popup menus in taskbar" is on the roadmap for further SEB 3 updates.
Are there concrete dates available for this functionality?
As this is an important feature for our quiz scenario we are still dependent on 2.4.1 ...
Looking forward your reply.
Best regards, Martin

**Comment by dbuechel** [comment]
We're currently quite busy with other priorities, thus there are no dates available for the additional resource features (apart from the rough approximation of 2022 / 2023 as indicated in our roadmap).

**Comment by danschlet** [comment]
Please note that it's very well possible to allow accessing specific web resources in all existing SEB versions using the URL Filter feature (which is also used by the Additional Resources feature). The difference is, that the UI to open such pages in additional browser windows/tabs would need to implemented in the exam web application (as some of our partners did).

We definitely want to reintroduce the Additional Resources feature, but as there are other priorities as mentioned (amongst others security and browser improvements and SEB Server and proctoring features), it just still will have to wait a bit.
